src/metabase/xrays/automagic_dashboards/core.clj

```diff
@@ -468,81 +473,26 @@
                                                (assoc group :title (or comparison_title title))))))
        (instantiate-metadata context available-metrics {}))))
 
-(defn affinities->viz-types
```

Comment:
I think we all agree the X-Ray code is super opaque so it's nice to remove a lot of it. Still not very easy to follow but at least there's less of it now.